slack.go

```diff
@@ -283,8 +283,8 @@ func (b *Bot) Start(manager JobManager) error {
 		},
 	})
 
-	slack.Command("build <from>", &slacker.CommandDefinition{
-		Description: "Create a new release image from one or more pull requests. The successful build location will be sent to you when it completes and then preserved for 12 hours.",
+	slack.Command("build <pullrequest>", &slacker.CommandDefinition{
+		Description: "Create a new release image from one or more pull requests. The successful build location will be sent to you when it completes and then preserved for 12 hours.  Example: `<build openshift/operator-framework-olm#68,operator-framework/operator-marketplace#396`",
```

Comment:
Small nit...
Did you intend to have a `<` in the example?

Reply:
nope, fixed thanks